src/cloudant/document.py

```diff
@@ -17,6 +17,17 @@ class CloudantDocument(dict):
     """
     _CloudantDocument_
 
+    JSON document object, used to manipulate the documents
+    in a couch or cloudant database. In addition to basic CRUD
+    style operations this provides a context to edit the document:
+
+    with document:
+        document['x'] = 'y'
+
+    :param cloudant_database: CouchDatabase or CloudantDatabase instance
```

Comment:
Yeah, and will probably rename CloudantDocument to Document since it works for both Couch and Cloudant